xml2rfc/writers/html.py

```diff
@@ -2808,7 +2808,14 @@ def render_uri(self, h, x):
     #    or
     # 
     #    [<a class="xref" href="#RFC1234">RFC1234</a>]
-    # 
+    #
+    # Regarding HTML class assignment:
+    #   Most links here will get the "xref" class.
+    #   Relative links to sections of citations get "relref", but not "xref".
+    #   Links to citations/references (like "[RFC2119]") get the "cite" class.
+    #   Links to sections (or appendices) of this document get the "internal" class.
+    #   "sref" links with default text (like "Section 1.3") get the "auto" class.
```

Comment:
What is sref?

Reply:
Sorry, I messed that up (it was a name I used for "internal" previously).

```suggestion
    #   Links with default text (like "Section 1.3") get the "auto" class.
```